packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -282,30 +282,27 @@ class PerformanceMonitoring {
 
   createTransactionPayload(transaction) {
     this.prepareTransaction(transaction)
-    var filtered = this.filterTransaction(transaction)
+    const filtered = this.filterTransaction(transaction)
     if (filtered) {
       return this.createTransactionDataModel(transaction)
     }
   }
 
   sendTransactions(transactions) {
-    var payload = transactions
-      .map(this.createTransactionPayload.bind(this))
-      .filter(function(tr) {
-        return tr
-      })
+    const payload = transactions.map(tr => this.createTransactionPayload(tr))
```

Comment:
I think the filter part is necessary to avoid having undefined in the array due to filtering!

```
      .filter(function(tr) {
        return tr
      })
```

Reply:
Good catch, we need to call filter first and map to avoid null values in array.